### ISSUE TITLE
User Interface default values from example.py

### ISSUE THREAD

**Issue by bahadirtortop** [andrewsegas/instaboost#9]
Hi,

Can you taking default values from example.py

**Comment by andrewsegas** [comment]
Yes, i know that is boring to fill every single time, but you can change the file parameters_ui.py with your default values, or if you are talking about parameters that is not in user interface you can change in instaboost.py
